frontend/src/lib/services/Libre311/Libre311.ts

```diff
@@ -265,6 +265,34 @@ export type ServiceRequest = z.infer<typeof ServiceRequestSchema>;
 export const GetServiceRequestsResponseSchema = z.array(ServiceRequestSchema);
 export type GetServiceRequestsResponse = z.infer<typeof GetServiceRequestsResponseSchema>;
 
+// ***************** Create Service *************** //
+
+// Create Service - Request Schema
+export const CreateServiceRequestSchema = z.object({
+	service_code: z.string(),
+	service_name: z.string(),
+	description: z.string(),
+	service_definition: z.string(),
+	group_id: z.number()
+});
+
+//  Create Service - Response Schema
+export const CreateServiceResponseSchema = z
+	.object({
+		id: z.number(),
+		jurisdiction_id: z.string(),
+		group_id: z.number()
+	})
+	.merge(ServiceSchema);
+
+//  Create Service - Request Type
+export type CreateServiceRequest = z.infer<typeof CreateServiceRequestSchema>;
```

Comment:
Imo the naming should be `CreateServiceParams`
We have the `createService` method and it expects params of type `CreateServiceParams`
That at least mirrors the `createServiceRequest` method which expects params of type `CreateServiceRequestParams`